cost-model/src/cost_model.rs

```diff
@@ -313,7 +346,9 @@ mod tests {
     use {
         super::*,
         itertools::Itertools,
-        log::debug,
```

Comment:
The tests here for the most part should be testing the cost model with _and_ without the new feature enabled